.env.example

```diff
@@ -14,5 +14,5 @@ export CLEAR_CACHE=0
 export TS_NODE_TRANSPILE_ONLY=1
 
 # Simulation parameters
-export FORK_BLOCK=14121827
+export FORK_BLOCK=13724056
```

Comment:
I know this is still a draft, but would prefer to leave this at block 14121827. I've started doing some benchmarks using that block which I'll add to the README soon (after this PR). This block also uses ~2M more gas than block 13724056, which is nice for benchmarking

Reply:
Works for me! I'll be sure to change it back before marking this for review.

Reply:
Hey @MicaiahReid, what's the status on this PR? Is this still needed, or do newer versions of ganache now automatically update the EVM timestamp when forming?

Reply:
Hi @mds1. Sorry, I've gotten pulled into other priorities, but yes, this still is needed. We do not currently plan to have forked blocks' timestamps be set to the fork block's time. We are working on some performance improvements to Ganache right now, so I'll circle back to this and get it merged once that's done.